modules/camera/camera_picamera2.py

```diff
@@ -3,6 +3,7 @@
 """
 
 import numpy as np
```

Comment:
Add a space after this to separate imported libraries from our own written libraries